File: app/src/main/java/com/globo/clappr/app/PlayerActivity.java
package com.globo.clappr.app;

import android.app.Activity;
import android.os.Bundle;
import android.os.Handler;
import android.view.Menu;
import android.view.MenuItem;

import com.globo.clappr.player.Player;

public class PlayerActivity extends Activity {

    private Player player;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_player);
        if (savedInstanceState == null) {
            player = Player.newInstance();
            getFragmentManager().beginTransaction()
                    .add(R.id.container, player)
                    .commit();
        }

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                player.load("http://bem.tv/superMisto.mp4");
            }
        }, 100);
    }


    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.player, menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        int id = item.getItemId();
        if (id == R.id.action_settings) {
            return true;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/kotlin/com/globo/clappr/app/PlayerActivity.kt
package com.globo.clappr.app

import android.app.Activity
import android.os.Bundle
import android.util.Log
import android.view.Menu
import android.view.MenuItem
import com.globo.clappr.Player
import com.globo.clappr.base.Callback
import com.globo.clappr.base.Event
import com.globo.clappr.base.Options
import com.globo.clappr.playback.ExoPlayerPlayback
import com.globo.clappr.plugin.Loader

class PlayerActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_player)


        val player = Player()
        Loader.registerPlayback(ExoPlayerPlayback::class)
        player.configure(Options(source = "http://clappr.io/highline.mp4", autoPlay = false))
        player.on(Event.WILL_PLAY.value, Callback.wrap {Log.i("PLAYER", "Will Play")})
        player.on(Event.PLAYING.value, Callback.wrap {Log.i("PLAYER", "Playing")})
        player.on(Event.DID_COMPLETE.value, Callback.wrap {Log.i("PLAYER", "Completed")})

        val fragmentTransaction = fragmentManager.beginTransaction()
        fragmentTransaction.add(R.id.container, player);
        fragmentTransaction.commit();

        player.play()
    }


    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        // Inflate the menu; this adds items to the action bar if it is present.
        menuInflater.inflate(R.menu.player, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        val id = item.itemId
        if (id == R.id.action_settings) {
            return true
        }
        return super.onOptionsItemSelected(item)
    }
}
